msteams-platform/apps-in-teams-meetings/build-tabs-for-meeting.md

```diff
@@ -176,6 +176,133 @@ TeamsJS is a rich library used to create Tabs using JavaScript. Use the latest T
+
+* Apps need to re-register for events such as `themeChange`, `focusEnter`, and so on, in the load handler. Teams client won't send any notifications to the app when cached. If your app requires notifications even when cached, caching might not be the right solution.
+
+* App caching is supported only in Teams desktop client. In Teams web client, even if the app registers load handlers, the app is removed from the cache after the unload sequence is completed.
+
+* Register the `load` and `beforeUnload` handlers early in your launch sequence. If the Teams client doesn’t identify these registrations before the user goes out of the app, the app isn't cached.
+
+* The Teams client invokes the `loadHandler` only after the `unload` sequence of the app is completed. For example, if a user launches tab A of your app and then launches tab B of the same app, tab B won't get the load signal until the tab A invokes the `readyToUnload` callback.
+
+* After an app moves to the cached state, it has a one-minute grace period to get the memory usage under the allowed threshold of 225 MB. If the app exceeds the threshold for more than a minute, the app is removed from the cache. The memory usage value of the app is the *workingSetSize* of the webview in the Electron [getappMetrics API](https://www.electronjs.org/docs/latest/api/app#appgetappmetrics).
+
+* Apps are cached on a per-window basis.
+
+* App caching isn't supported for the meeting stage or Task module contexts, because these can be opened on top of the tab and the same webview can't be used to render the content in the tab and the Task module.
+
+* App caching happens on a per app (not on a per tab) basis within the same window.
+
+* Apps are expected to sleep when cached (use minimal compute or network resources and minimizes SDK requests). All the register handlers and the following SDK requests are allowed when the app is cached:
+
+  * `initialize`
+  * `notifyappLoaded`
+  * `notifySuccess`
+  * `notifyFailure`
+  * `notifyExpectedFailure`
+  * `getContext`
+  * `getAuthToken`
+  * `readyToUnload`
+  * `getConfig/getSettings`
+
+* Register only the `beforeUnload` handler if your app doesn't require app caching but needs time to safely save state (if you want to ensure that going away from your app doesn't cause the app content to be abruptly removed from the Document Object Model (DOM)). If the app isn't registered for the `load` event, it's removed from the DOM after the unload flow completes.
+
+* For app caching to support only in meetings, register the `load` or `beforeUnload` handlers if the context is `meetingSidePanel`. Meetings is the first surface to support app caching. Even if app caching in other contexts such as `channels` and `chat` happens to work, it isn't officially supported at this point and is subject to change.
```

Comment:
delete this bit..
"Even if app caching in other contexts such as `channels` and `chat` happens to work, it isn't officially supported at this point and is subject to change."
Change "Meetings is the first surface to support app caching"
to
Side Panel in Meetings  is the first context to officially support app caching

Reply:
remove the last line

Reply:
Removed as suggested